pkg/kubelet/apis/podresources/types.go

```diff
@@ -35,6 +35,7 @@ type DevicesProvider interface {
 // PodsProvider knows how to provide the pods admitted by the node
 type PodsProvider interface {
 	GetPods() []*v1.Pod
+	GetPodByName(namespace, name string) (*v1.Pod, bool)
```

Comment:
ok, this is already implemented and we just need to expose it in the `PodsProvider` interface.

Reply:
yes that true